### ISSUE TITLE
[Bug]: Stretchly has an color issue with version 1.8.0

### ISSUE THREAD

**Issue by khanhtuongle** [hovancik/stretchly#1043]
### Version

- [X] I'm using version 1.8.0

### Known issues

- [X] I've checked Known issues

### Existing issues

- [X] I've checked Existing issues

### What operating system are you using?

Windows

### Operating System Version

windowns 11, 21H2

### Reproduction steps

1.

### Expected Behavior

I just update the 1.8.0 version and now it is like this:

![image](https://user-images.githubusercontent.com/89136898/142789343-8e8beba0-9eb7-4d29-8a79-ef6fe00bc893.png)

### Actual Behavior

I hope you can help me to resolve this issue, thank you!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KarlManong** [comment]
I got the same issue on macos monterey m1 version

**Comment by raphaeltao** [comment]
> I got the same issue on macos monterey m1 version

so do I
on new mbp 14

**Comment by Josephfilbert** [comment]
Same issue with Windows 10, no matter what color is chosen, it results in white text on white background. Decided to downgrade to previous version 1.7.0 and works fine.

**Comment by khanhtuongle** [comment]
> Same issue with Windows 10, no matter what color is chosen, it results in white text on white background. Decided to downgrade to previous version 1.7.0 and works fine.

Yes, I don't know why but version 1.7.0 works fine

**Comment by thekumarpraveen** [comment]
I am also facing the same issue. Reverted to V1.07. Hope the developer attends this. Stretchly is really a great free app.

**Comment by CCMEOW** [comment]
same issue on mac(os catalina version 10.15.7), stretchly version 1.8.0.

**Comment by BrunoBernardino** [comment]
Same issue on Zorin OS 16 (Ubuntu 20.04). If you installed via `snap` and are looking to revert to 1.7.0, `sudo snap revert stretchly` should be enough. If you want to temporarily disable auto-updates, you can learn more at https://snapcraft.io/docs/keeping-snaps-up-to-date

Changing colors or toggling transparency yields no change in the break screen.

Here's the error I see when initializing 1.8.0 (no error when starting the break screens):

```
› Stretchly: starting Do Not Disturb monitoring
[30752:1122/092838.437553:ERROR:sandbox_linux.cc(376)] InitializeSandbox() called with multiple threads in process gpu-process.
[30563:1122/092838.455033:ERROR:object_proxy.cc(642)] Failed to call method: org.freedesktop.login1.Manager.Inhibit: object_path= /org/freedesktop/login1: org.freedesktop.DBus.Error.AccessDenied: An AppArmor policy prevents this sender from sending this message to this recipient; type="method_call", sender=":1.273" (uid=1000 pid=30563 comm="/snap/stretchly/7/stretchly --no-sandbox --enable-" label="snap.stretchly.stretchly (enforce)") interface="org.freedesktop.login1.Manager" member="Inhibit" error name="(unset)" requested_reply="0" destination="org.freedesktop.login1" (uid=0 pid=1247 comm="/lib/systemd/systemd-logind " label="unconfined")
```

I do not see any new snap permissions to allow. I usually don't give it permission to play sounds, but even enabling that didn't make the error go away.

**Comment by svaneke** [comment]
Same here, macOS 10.14.6 with Stretchly 1.8.0.

**Comment by JorisDebonnet** [comment]
Hi @hovancik, seems like an urgent issue, because it seems to be appearing on all platforms ☺

**Comment by hovancik** [comment]
Interesting, for me all works fine on MacOS 11.6.1, Windows 10 and Debian with 1.8.

**Comment by hovancik** [comment]
Does log shows any errors?

**Comment by 12people** [comment]
@hovancik Not on macOS Big Sur.
Just reported the same bug at #1044 — seems like a duplicate. This bug should really be renamed, as its title is really non-specific and misleading.

**Comment by JorisDebonnet** [comment]
On Windows 10, `main.log` doesn't show anything special either...
The mainColor is also correctly stored in config.json.

(I do use it only on my second screen, in case that matters. Well - just tested it, and even if I turn off second screen and only keep it on the laptop's native screen, the issue remains the same.)

**Comment by hovancik** [comment]
@12people thanks, your issue pointed me in right direction, can reproduce that.

**Comment by jjstickel** [comment]
Same washed-out color behavior on Ubuntu 21.10. Only "Snow white" looks reasonable, although I think it is because the text is in black, not because the background color is necessarily correct.

And yes, please add something about "color" to the title of the issue"

**Comment by imkarimkarim** [comment]
i have the same issue on Ubuntu 20.04.3 LTS

**Comment by BrunoBernardino** [comment]
I can confirm it's fixed for me on `1.8.1`. I just had to run `sudo snap refresh stretchly` to get it updated because I had reverted it before. Thanks for the quick fix @hovancik !
